docs/_index.md

```diff
@@ -14,7 +14,6 @@ Crossplane is an open source Kubernetes add-on that transforms your cluster into
 a **universal control plane**. Crossplane enables platform teams to assemble
 infrastructure from multiple vendors, and expose higher level self-service APIs
 for application teams to consume, without having to write any code.
-
```

Comment:
@plumbis is dropping this line-break intentional?

Reply:
Not sure how that happened. Re-added.